deploy/kube-config/influxdb/grafana-service.yaml

```diff
@@ -2,16 +2,17 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    kubernetes.io/cluster-service: 'true'
```

Comment:
This is just me being a newbie to Kubernetes :-) - I didn't realize these had any relevance beyond the deployment -- I will add both of these labels back.